pages/1.9/release-notes/1.9.9/index.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.9.9
+title: Release Notes for 1.9.9
+menuWeight: 0
+excerpt: Release notes for DC/OS 1.9.9
+---
+
+DC/OS 1.9.9 was released on July 10, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.9.9/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.9.9 includes the following:
+
+- Updated to Mesos 1.2.x ([change log](https://github.com/dcos/dcos/blob/1.9.9/packages/mesos/buildinfo.json)).
+- Updated to Marathon 1.4.11 ([change log](https://github.com/mesosphere/marathon/releases/tag/v1.4.11)).
+- Updated to Metronome 0.3.2 ([change log](https://github.com/dcos/metronome/releases/tag/v0.3.2)).
+
+# Issues Fixed in DC/OS 1.9.9
+
+- COPS-2041/DCOS-21467 - DC/OS UI: Fixed labels reducer to accept empty strings.
+- COPS-3180 - Fixed odd exit behavior for tasks that use a volume mount (via rexray). [enterprise type="inline" size="small" /]
+- DCOS-14199 - Consolidated the Exhibitor bootstrapping shortcut by atomically reading and writing the ZooKeeper PID file. [enterprise type="inline" size="small" /]
+- DCOS-21557/DCOS_OSS-2367 - Updated cURL to version 7.59.
```

Comment:
This might be a notable change?

Reply:
See https://docs.mesosphere.com/1.11/release-notes/1.11.1/#issues-fixed-in-dcos-1111. 
DCOS-21557 was reviewed and approved under "Issues fixed" section.

Reply:
The cURL upgrade was recently found to have some configuration impacts to users using https proxies - they must change their configuration to work properly with this new cURL. So I would agree with Judith that this is a 'notable' change.

Reply:
Do we have any more detail on what exactly they'll need to modify? It would be really great to include that in the notes since I think lots of people use https proxies.

Reply:
@cneth @orsenthil @jgehrcke ???

Reply:
> The cURL upgrade was recently found to have some configuration impacts to users using https proxies - they must change their configuration to work properly with this new cURL. So I would agree with Judith that this is a 'notable' change.

what do you refer to @cneth? If it's https://jira.mesosphere.com/browse/COPS-3470 then "the curl upgrade" from 7.5x to 7.59 is unrelated to COPS-3470.

This curl update belongs into the "security updates" section of the release notes.

Reply:
@jgehrcke moving "DCOS-21557/DCOS_OSS-2367 - Updated cURL to version 7.59." to Security Enhancements is correct. 
Do we have any more detail on what exactly they'll need to modify?>>>>Judith's question.

Reply:
Do we mention COPS tickets in the RN at all?

Reply:
@marcomonaco yes we do.

Reply:
Repeating; 

This curl update belongs into the "security updates" section of the release notes.

Reply:
Moved curl update to Security updates section.